src/api/auth.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+const API = 'http://localhost:4000/api'
```

Comment:
Is this the right URL for prod?

Reply:
I'm using the local host right now, I was going to change the URL once I finish the base for the app, is that ok?

Reply:
No, I haven't deployed it yet, should I deploy it first?

Reply:
Sure thing, we can discuss it at stand up :)